metadata_lookup.ipynb

```diff
@@ -0,0 +1,995 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # they didnt reply :( need help
</pre><p>Can you forward me the context on Slack, please? I'll annoy them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>